### ISSUE TITLE
--allman false doesn't work for computed properties

### ISSUE THREAD

**Issue by subdan** [nicklockwood/SwiftFormat#554]
Hello, @nicklockwood.

I've found an issue with [`braces`](https://github.com/nicklockwood/SwiftFormat/blob/master/Rules.md#braces) rule. Option `--allman false` doesn't work for computed properties.

Input
```swift
class Man
{
    let name: String
    var descriptionProperty: String
    {
        return name
    }
    func descriptionFunc() -> String
    {
        return name
    }
}
```

Expected output
```swift
class Man {
    let name: String
    var descriptionProperty: String {
        return name
    }
    func descriptionFunc() -> String {
        return name
    }
}
```

Actual output
```swift
class Man {
    let name: String
    var descriptionProperty: String
    {
        return name
    }
    func descriptionFunc() -> String {
        return name
    }
}
```

**Comment by subdan** [comment]
In `Rules.swift` file on 1343 line I've found this code:
```
            guard !formatter.isStartOfClosure(at: i) else {
                return
            }
```
Computer property isn't a closure. Is it a bug in `isStartOfClosure` implementation?

**Comment by subdan** [comment]
I've found another issue. Option --allman false doesn't work with `init`.
Input
```swift
class Man
{
    init()
    {
        print("init")
    }
}
```

Expected output
```swift
class Man {
    init() {
        print("init")
    }
}
```

Actual output
```swift
class Man {
    init()
    {
        print("init")
    }
}
```

**Comment by nicklockwood** [comment]
@subdan thanks for the report! I think your guess about the root cause seems likely - I'll investigate further.

**Comment by nicklockwood** [comment]
@subdan this is now fixed on the [develop](https://github.com/nicklockwood/SwiftFormat/tree/develop) branch.

**Comment by subdan** [comment]
Wow! You are super fast. Thanks!

**Comment by nicklockwood** [comment]
@subdan fixed in 0.44.2.
